torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -712,6 +713,7 @@ class TensorExprFuser {
   // Merge fusible nodes into subgraphs in prim::TensorExprGroup nodes.
   void createFusionGroups(Block* block) {
     bool any_changed = true;
+    parseTENotFuseOption();
```

Comment:
This function initializes the `operators_not_to_fuse` member variable, right?  I'd recommend calling it in the constructor rather than here in one of the member functions.